### PR TITLE
Add Japanese translation file.

### PR DESCRIPTION
**Pull request by mayukojpn**

### CODE REVIEW

File: languages/tgmpa-ja.po [hunk 1]
```diff
@@ -0,0 +1,470 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: TGM Plugin Activation\n"
+"POT-Creation-Date: 2016-05-19 02:57+0200\n"
+"PO-Revision-Date: 2017-08-09 11:13+0900\n"
+"Last-Translator: Juliette Reinders Folmer <wpplugins_nospam@adviesenzo.nl>\n"
```
Comment (GaryJones) [inline languages/tgmpa-ja.po]: Please update this to your details @mayukojpn :-)

File: languages/tgmpa-ja.po [hunk 2]
```diff
@@ -0,0 +1,470 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: TGM Plugin Activation\n"
```
Comment (GaryJones) [inline languages/tgmpa-ja.po]: Please add the version number in.

**Review by GaryJones — CHANGES_REQUESTED** [review]
Couple of header strings to change please, then good to go.

File: languages/tgmpa-ja.po (hunk 1 above)
Comment (mayukojpn) [reply languages/tgmpa-ja.po]: [Done](https://github.com/TGMPA/TGM-Plugin-Activation/pull/683/commits/b4e0f9f96d143b457d5f192b3ed65766ab75fa4d)!

**Review by GaryJones — APPROVED** [review]
Thank you @mayukojpn !

**Pull request merged**